mitodl/edx-sysadmin
# Basic architecture for edx-sysadmin plugin

### Related Ticket:
https://github.com/mitodl/edx-platform/issues/245

### How to test:
Clone this branch on your local devstack (i.e in `src` folder of edx)
Install it through pip (you might have to restart devstack after installing this plugin)
Go to `/sysadmin/` (make sure that the `ENABLE_SYSADMIN_DASHBOARD` feature flag is turned off).


### Screenshots:
![image](https://user-images.githubusercontent.com/42243411/110773206-4cbee080-827e-11eb-8c5c-4ca424214599.png)
